app/components/reactions/index.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {addReaction, getReactionsForPost, removeReaction} from 'mattermost-redux/actions/posts';
+import {makeGetReactionsForPost} from 'mattermost-redux/selectors/entities/posts';
+
+import {getTheme} from 'app/selectors/preferences';
+
+import Reactions from './reactions';
+
+const getReactionsForPostSelector = makeGetReactionsForPost();
```

Comment:
@jarredwitt wondering if this should be wrapped in a `makeMapStateToProps` function